extension/src/test/XLIFFTypes.test.ts

```diff
@@ -265,28 +260,28 @@ export function GetSmallXliffXmlWithHtmlTag(): string {
 }
 
 
-export function getSmallFormattedXliffXml(): string {
-  return `<?xml version="1.0" encoding="utf-8"?>
-<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
-  <file datatype="xml" source-language="en-US" target-language="sv-SE" original="AlTestApp">
-    <body>
-      <group id="body">
-        <trans-unit id="Table 2328808854 - NamedType 12557645" size-unit="char" translate="yes" xml:space="preserve">
-          <source>This is a test ERROR in table</source>
-          <target>This is a test ERROR in table</target>
-          <note from="Developer" annotates="general" priority="2"></note>
-          <note from="Xliff Generator" annotates="general" priority="3">Table MyTable - NamedType TestErr</note>
-        </trans-unit>
-        <trans-unit id="Page 2931038265 - NamedType 12557645" size-unit="char" translate="yes" xml:space="preserve">
-          <source>This is a test ERROR</source>
-          <target>This is a test ERROR</target>
-          <note from="Developer" annotates="general" priority="2"></note>
-          <note from="Xliff Generator" annotates="general" priority="3">Page MyPage - NamedType TestErr</note>
-        </trans-unit>
-      </group>
-    </body>
-  </file>
-</xliff>`;
+export function GetSmallFormattedXliffXml(): string {
+  return `<? xml version = "1.0" encoding = "utf-8" ?>
+    <xliff version="1.2" xmlns = "urn:oasis:names:tc:xliff:document:1.2" xmlns: xsi = "http://www.w3.org/2001/XMLSchema-instance" xsi: schemaLocation = "urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd" >
```

Comment:
Yes it's super strange. Something got messed up in merge from master. It took me a few tries to get the correct formatting to persist. It switch back after when leaving the file for some reason. It feels like a possible VS Code or Git bug, i'd blame VS Code befor Git tho ;)

What's your setting for spaces and line endings on this file? (bottom right corner)
![image](https://user-images.githubusercontent.com/17023248/101459162-2f554000-3938-11eb-9421-f99593a63dd5.png)

Reply:
default values, 4 spaces and CRLF (of course!). Don't think the line endings matter, though, since it's configured to commit as LF.
![image](https://user-images.githubusercontent.com/23032645/101462362-70e7ea00-393c-11eb-87d7-3adfed040261.png)

We should add the Tab Size setting in the workspace, I guess :)

Btw, I don't see the strange formatting issue - so I guess it's a VSCode on Linux bug/feature :)